e2e/cypress/integration/enterprise/cloud/billing/banner_trial_spec.js

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @cloud_only @cloud_trial
+// Skip:  @headless @electron // run on Chrome (headed) only
+
+describe('Banner - Subscription scenarios', () => {
+    before(() => {
+        // * Check if server has license for Cloud
+        cy.apiRequireLicenseForFeature('Cloud');
+    });
+
+    describe('Banner - UI validations', () => {
+        it('MM-T4422 Cloud trial prompts for admin login', () => {
+            // # Login as sysadmin
+            cy.apiAdminLogin();
+
+            cy.apiInitSetup().then(({team}) => {
+                // # Go to town square
+                cy.visit(`/${team.name}/channels/town-square`);
+                cy.waitForResources();
```

Comment:
What are we waiting for when we call this with no arguments? And why do we need to wait for background requests when we could just wait for the app to render like other tests do?

Reply:
Idea here is to wait for network idle and all the background http calls (scrpti,xhr,fetch) to get complete so that I can confidently run the other commands of cypress.